### DISCUSSION TITLE
posts/vim-basic-commands/

### DISCUSSION THREAD

**Discussion by giscus** [m4xshen/m4xshen.dev#5] [Announcements]
# posts/vim-basic-commands/

Master the essential Vim commands for efficient text editing. Learn Normal, Insert, and Command-line modes, navigation, editing, and more in this comprehensive Vim tutorial.

https://m4xshen.dev/posts/vim-basic-commands/

**Comment by ghost** [comment]
Thank you for the wonderful post.
